src/components/InputLabel/InputLabel.module.css

```diff
@@ -0,0 +1,30 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # INPUT LABEL
+\*------------------------------------*/
+
+/**
+ * Text labelling the checkbox.
+ */
+.label {
+  color: var(--eds-theme-color-text-neutral-strong);
+}
+
+/**
+ * Disabled variant of the checkbox.
+ */
+.label--disabled {
+  color: var(--eds-theme-color-text-disabled);
+}
+
+/**
+ * Checkbox size variants.
+ * 1) Centers the first line of the medium label with the checkbox.
```

Comment:
Is this comment relevant here? I see the same line in `src/components/CheckboxLabel/CheckboxLabel.module.css`.

Reply:
Crap, I'll update all the comments referring to `checkbox` in this file